.buildkite/pipeline.yml

```diff
@@ -1,28 +1,55 @@
+env:
+  ANKA_MOJAVE_TEMPLATE: "10.14.4_6C_14G_40G"
+  ANKA_TEMPLATE_TAG: "clean::cicd::git-ssh::nas::brew::buildkite-agent-3.11.2"
+  CHECKSUMABLE: "scripts/eosio_build*"
+  MAC_TAG: "eosio_2-4"
+
 steps:
-  - command: | # macOS Mojave Build
-      echo "--- Creating symbolic link to job directory :file_folder:"
-      sleep 5
-      ln -s "$(pwd)" /data/job
-      cd /data/job
-      echo "+++ Building :hammer:"
-      ./scripts/eosio_build.sh -y -P -m
-      echo "--- Compressing build directory :compression:"
-      tar -pczf build.tar.gz build
-      if [[ ! -f build.tar.gz ]]; then echo 'ERROR: No build.tar.gz artifact found!' && exit 1; fi
-    label: ":darwin: Mojave Build"
+
+  - trigger: "mac-anka-fleet"
+    label: ":anka: Ensure Mojave Anka Template Tag Exists"
+    branches: "*"
+    async: false
+    build:
+      branch: "master"
+      env:
+        REPO: "${BUILDKITE_REPO}"
+        REPO_BRANCH: "${BUILDKITE_BRANCH}"
+        CHECKSUMABLE: "${CHECKSUMABLE}"
+        TEMPLATE: "${ANKA_MOJAVE_TEMPLATE}"
+        TEMPLATE_TAG: "${ANKA_TEMPLATE_TAG}"
+        TAG_COMMANDS: "CLONED_REPO_DIR/scripts/eosio_build.sh -y -P -m" # CLONED_REPO_DIR IS REQUIRED and is where the repo is always cloned into
+        PROJECT_TAG: "${MAC_TAG}"
+
+  - wait
+
+  ############################
+  # BUILDING #################
+  ############################
+
+  - label: ":darwin: [Darwin] Mojave Build"
+    command:
+      - "git clone git@github.com:EOSIO/eos.git -b ${BUILDKITE_BRANCH} eos && cd eos && git submodule update --init --recursive && ./scripts/eosio_build.sh -y -P -m"
+      - "tar -pczf /Network/NAS/MAC_FLEET/BUILDKITE/artifacts/${ANKA_MOJAVE_TEMPLATE}-${BUILDKITE_PIPELINE_SLUG}-${BUILDKITE_BUILD_ID}.tar.gz eos"
+    plugins:
+      chef/anka#v0.4.4:
+        no-volume: true
+        inherit-environment-vars: true
+        vm-name: $ANKA_MOJAVE_TEMPLATE
+        vm-registry-tag: "${ANKA_TEMPLATE_TAG}::${MAC_TAG}"
+        modify-cpu: 12
+        modify-ram: 24
+        always-pull: true
+        debug: true
+        wait-network: true
     agents:
-      - "role=builder-v2-2"
-      - "os=mojave"
-    artifact_paths: "build.tar.gz"
+      - "queue=mac-anka-large-node-fleet"
     timeout: 120
 
-  - command: | # Amazon Linux 2 Build
-        echo "+++ :hammer: Building"
-        ./scripts/eosio_build.sh -y -P -m
-        echo "--- :compression: Compressing build directory"
```

Comment:
Please don't delete these types of print statements.
Without this print statement, it will look like the agent has hanged.